src/components/Card/Card.jsx

```diff
@@ -220,8 +238,7 @@ const Card = ({
     </StyledToolbar>
   ) : (
     <StyledToolbar>
-      {// TODO: if we keep this, expose capability under prop
-      false && timeBoxSelection}
+      {mergedAvailableActions.range && timeBoxSelection}
```

Comment:
I think we should also hide the range picker if the Card is in the editable state for now...

Reply:
yep, that's covered by the ternary branch on `isEditable` -- if it's in editable state the range picker (and expansion button) won't be rendered